test_settings.py

```diff
@@ -33,8 +33,9 @@ def root(*args):
     "django.contrib.sessions",
     "django.contrib.staticfiles",
     # Admin
-    #    'django.contrib.admin',
-    #    'django.contrib.admindocs',
+    'django.contrib.admin',
+    'django.contrib.admindocs',
+    'debug_toolbar',
```

Comment:
If I comment 'django.contrib.admin' and 'django.contrib.admindocs' I get de following error in the tests:
```
FAILED tests/openedx_tagging/core/tagging/test_views.py::TestTaxonomyViewSet::test_create_taxonomy_1__None__403_ - LookupError: No installed app with label 'admin'.
```

The `django-debug-toolbar` was because I was getting an error in the following line in the CI:
https://github.com/open-craft/openedx-learning/blob/107c1f6db4c9758186b09011d513a899c14bd5e1/projects/urls.py#L13

CI Error: https://github.com/open-craft/openedx-learning/actions/runs/5599880930/jobs/10241446948?pr=4#step:7:450

The local test runs just fine.